extensions/tmtv/src/utils/hpViewports.ts

```diff
@@ -128,6 +128,7 @@ const ptAXIAL = {
         target: false,
         options: {
           syncColormap: false,
+          syncInvertState: false,
```

Comment:
I remember this caused something else to get broken, can you see git blame who removd it and why?

Reply:
ok, but for some reason if its not set, it defaults to true, why causes that stupid bug that has been there for few months now in tmtv